lib/promscrape/client.go

```diff
@@ -103,6 +104,10 @@ func newClient(ctx context.Context, sw *ScrapeWork) *client {
 		isTLS = pu.Scheme == "https"
 		if isTLS {
 			tlsCfg = sw.ProxyAuthConfig.NewTLSConfig()
+			flagValue, tlsVerificationFlagFound := os.LookupEnv("skip_tls_verification")
```

Comment:
Env parameter instead of a config field as this makes it easy to expose it as a CF Template parameter